src/main/java/com/example/usermanagement/controller/UserController.java

```diff
@@ -15,43 +13,43 @@
 
 
 @RestController
-@RequestMapping("/users")
+@RequestMapping("api/v1/users")
 public class UserController {
 
 
     @Autowired
     UserService userService;
 
-    //Create user
-    @PostMapping()
-    public ResponseEntity<Response> createUser(@Valid @RequestBody CreateUserdto createUserdto){
-        return ResponseEntity.status(HttpStatus.CREATED).body(userService.create(createUserdto));
-    }
-
     //Update a user
-    @PutMapping("/{userId}")
-    public ResponseEntity<Response> updateUser(@PathVariable Long userId, @RequestBody UpdateUserDto updateUserDto) {
-        return ResponseEntity.status(HttpStatus.OK).body(userService.update(updateUserDto, userId));
+    @PutMapping("/{username}")
+    public ResponseEntity<Response> updateUser(@PathVariable String username, @RequestBody UpdateUserDto updateUserDto) {
+        return ResponseEntity.status(HttpStatus.OK).body(userService.update(updateUserDto, username));
```

Comment:
Why this change?

Reply:
Because we are not letting user know their userId when they create a user, it only shows username,firstName and lastname there. So for that reason i have used username here.